1-JavaScript/Week2/homeworkSolition/ex4-sol-shoppingCart.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+const shoppingCart = ['bananas', 'milk'];
+
+function addToShoppingCart(groceryItem) {
+if(groceryItem !== undefined){shoppingCart.push(groceryItem)}
+if(shoppingCart.length > 3 ){shoppingCart.shift()}
+  return `You bought ${shoppingCart.join(", ")}!`
+}
```

Comment:
Fantastic job rewriting the code! 💪 

I would suggest to add some more spaces in between to make it a bit more readable, it's a little bit hard to read when everything is next to each other 🙂 

```suggestion
function addToShoppingCart(groceryItem) {
   if(groceryItem !== undefined) {
      shoppingCart.push(groceryItem)
   }
   
   if(shoppingCart.length > 3 ) {
      shoppingCart.shift()
    }
    
    return `You bought ${shoppingCart.join(", ")}!`
}
```